.docker/Dockerfile

```diff
@@ -1,68 +1,82 @@
+    tar xfJ aravis-0.8.30.tar.xz && \
+    rm aravis-0.8.30.tar.xz && \
+    cd aravis-0.8.30 && \
+    meson setup build && \
+    cd build && \
+    ninja && \
+    checkinstall --pkgname aravis --pkgversion 0.8.30 --requires="libglib2.0-dev, libxml2-dev, libusb-1.0-0-dev" ninja install
 
 SHELL ["/bin/bash", "-c"]
-RUN wget https://software.micro-epsilon.com/scanCONTROL-Linux-SDK-${SCANCONTROL_SDK_VERSION//./-}.zip && \
-    unzip scanCONTROL-Linux-SDK-${SCANCONTROL_SDK_VERSION//./-}.zip && \
+RUN wget https://software.micro-epsilon.com/scanCONTROL-Linux-SDK-${SCANCONTROL_SDK_VERSION//./-}.zip -O scanCONTROL-Linux-SDK-${SCANCONTROL_SDK_VERSION//./-}.zip && \
+    unzip scanCONTROL-Linux-SDK-${SCANCONTROL_SDK_VERSION//./-}.zip -d scanCONTROL-Linux-SDK/ && \
     rm scanCONTROL-Linux-SDK-${SCANCONTROL_SDK_VERSION//./-}.zip && \
-    cd /scanCONTROL\ Linux\ SDK\ ${SCANCONTROL_SDK_VERSION}/libmescan/ && \
+    cd scanCONTROL-Linux-SDK/libmescan/ && \ 
     meson builddir && \
     cd builddir && \
     ninja && \
-    checkinstall --pkgname mescan --pkgversion ${SCANCONTROL_SDK_VERSION} --requires="aravis \(\>= 0.6.0\)" ninja install && \
+    checkinstall --pkgname mescan --pkgversion ${SCANCONTROL_SDK_VERSION} --requires="aravis \(\>= 0.8.0\)" ninja install && \
     ldconfig && \
-    cd "/scanCONTROL Linux SDK ${SCANCONTROL_SDK_VERSION}/libllt/" && \
+    cd /scanCONTROL-Linux-SDK/libllt && \
     meson builddir && \
     cd builddir && \
     ninja && \
-    checkinstall --pkgname llt --pkgversion ${SCANCONTROL_SDK_VERSION} --requires="mescan \(\>= ${SCANCONTROL_SDK_VERSION}\),aravis \(\>= 0.6.0\)" ninja install
+    checkinstall --pkgname llt --pkgversion ${SCANCONTROL_SDK_VERSION} --requires="mescan \(\>= ${SCANCONTROL_SDK_VERSION}\),aravis \(\>= 0.8.0\)" ninja install
 
 RUN mkdir /library_pkgs && \
-    mv /aravis-0.6.4/aravis_0.6.4-1_amd64.deb /library_pkgs && \
-    mv "/scanCONTROL Linux SDK ${SCANCONTROL_SDK_VERSION}/libmescan/builddir/mescan_${SCANCONTROL_SDK_VERSION}-1_amd64.deb" /library_pkgs && \
-    mv "/scanCONTROL Linux SDK ${SCANCONTROL_SDK_VERSION}/libllt/builddir/llt_${SCANCONTROL_SDK_VERSION}-1_amd64.deb" /library_pkgs
+    mv /aravis-0.8.30/build/aravis_0.8.30-1_amd64.deb /library_pkgs && \
+    mv "/scanCONTROL-Linux-SDK/libmescan/builddir/mescan_${SCANCONTROL_SDK_VERSION}-1_amd64.deb" /library_pkgs && \
+    mv "/scanCONTROL-Linux-SDK/libllt/builddir/llt_${SCANCONTROL_SDK_VERSION}-1_amd64.deb" /library_pkgs
 
-FROM ros:melodic-ros-core
+FROM ros:noetic-ros-core
 ARG SCANCONTROL_SDK_VERSION
 
-RUN apt-get update && \
-    apt-get install -y \
+RUN apt-get update && apt-get install -y --no-install-recommends\
       intltool \
       pkg-config \
     && rm -rf /var/lib/apt/lists/*
 
 COPY --from=build ["/library_pkgs", "/library_pkgs"]
 
 RUN apt-get update && \
-    apt install -y /library_pkgs/aravis_0.6.4-1_amd64.deb && \
+    apt install -y /library_pkgs/aravis_0.8.30-1_amd64.deb && \
```

Comment:
It's preferable to use `apt-get` rather than `apt` in scripts - it has a more reliable command line interface.

I think you can do all of the packages in one install command too (but please test it works if you change it).

Finally, if these are the only files in `/library_pkgs/`, then you could consider using
`apt-get install /library_pkgs/*` to make the command shorter. Obviously if there are additional files in there that you don't want, or the contents are subject to change, then don't do this. :)